src/libraries/System.Linq.Expressions/src/System/Linq/Expressions/BinaryExpression.cs

```diff
@@ -11,9 +11,58 @@
+    /// |<see cref="System.Linq.Expressions.ExpressionType.OrElse" />|<see cref="System.Linq.Expressions.Expression.OrElse(Expression, Expression)" />|
+    /// ## Comparison Operations
+    /// |Node Type|Factory Method|
+    /// |---------------|--------------------|
+    /// |<see cref="System.Linq.Expressions.ExpressionType.Equal" />|<see cref="System.Linq.Expressions.Expression.Equal(Expression, Expression)" />|
+    /// |<see cref="System.Linq.Expressions.ExpressionType.NotEqual" />|<see cref="System.Linq.Expressions.Expression.NotEqual(Expression, Expression)" />|
+    /// |<see cref="System.Linq.Expressions.ExpressionType.GreaterThanOrEqual" />|<see cref="System.Linq.Expressions.Expression.GreaterThanOrEqual(Expression, Expression)" />|
+    /// |<see cref="System.Linq.Expressions.ExpressionType.GreaterThan" />|<see cref="System.Linq.Expressions.Expression.GreaterThanOrEqual(Expression, Expression)" />|
+    /// |<see cref="System.Linq.Expressions.ExpressionType.LessThan" />|<see cref="System.Linq.Expressions.Expression.LessThan(Expression, Expression)" />|
+    /// |<see cref="System.Linq.Expressions.ExpressionType.LessThanOrEqual" />|<see cref="System.Linq.Expressions.Expression.LessThanOrEqual(Expression, Expression)
+    /// " />|
+    /// ## Coalescing Operations
+    /// |Node Type|Factory Method|
+    /// |---------------|--------------------|
+    /// |<see cref="System.Linq.Expressions.ExpressionType.Coalesce" />|<see cref="System.Linq.Expressions.Expression.Coalesce(Expression, Expression)" />|
+    /// ## Array Indexing Operations
+    /// |Node Type|Factory Method|
+    /// |---------------|--------------------|
+    /// |<see cref="System.Linq.Expressions.ExpressionType.ArrayIndex" />|<see cref="System.Linq.Expressions.Expression.ArrayIndex(Expression, Expression)" />|
+    /// In addition, the <see cref="System.Linq.Expressions.Expression.MakeBinary(ExpressionType, Expression, Expression)" /> methods can also be used to create a <see cref="System.Linq.Expressions.BinaryExpression" />. These factory methods can be used to create a <see cref="System.Linq.Expressions.BinaryExpression" /> of any node type that represents a binary operation. The parameter of these methods that is of type <see cref="System.Linq.Expressions.Expression.NodeType" /> specifies the desired node type.</remarks>
+    /// <example>The following example creates a <see cref="System.Linq.Expressions.BinaryExpression" /> object that represents the subtraction of one number from another.
+    /// :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_CLR_System/system.Linq.Expressions.Expression/CS/Expression.cs" id="Snippet8":::
+    /// :::code language="vb" source="~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.Linq.Expressions.Expression/VB/Expression.vb" id="Snippet8":::</example>
```

Comment:
I think the `:::code` syntax is incorrect; @carlossanlop to verify though. I believe these [need to be converted](https://github.com/carlossanlop/DocsPortingTool/blob/main/BackportInstructions.md#user-content-instructions:~:text=link%20it%20as%20a%20code%20file) to `[!code-csharp]` syntax.

If so, there are a lot of these throughout the PR and should be fixed in the porting tool.

Reply:
Some of these blocks define additional attributes, for example `interactive`: https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Linq/Enumerable.xml#L117

It's not clear to me how that could be mapped to `code-csharp` syntax accurately. Any hints @carlossanlop?

Reply:
Hopefully we can leave the syntax as `:::code` because @mairaw [just converted it](https://github.com/dotnet/dotnet-api-docs/pull/6609) all a couple weeks ago. Is there a problem leaving it in that format? That way we preserve interactive info too.

Reply:
The tool detected an `## Example` header and separated the code examples from the remarks section. What the tool didn't do, is detect the new `:::code` sections, which should've been preserved inside a markdown section.

@gewarren @mairaw correct me if I'm wrong: in dotnet-api-docs, do the :::code sections only work if they are inside a remarks section? Or do they also work if they are in plain xml, like here?

Reply:
I believe the ::: syntax is Markdown-specific. @tdykstra is there a way to specify interactive with `![]()` style code snippets? I don't even see that syntax mentioned in the [contributor guide](https://review.docs.microsoft.com/en-us/help/contribute/code-in-docs?branch=master#snippet-syntax-reference) anymore.

Reply:
There is, but I believe the old format is Markdown-specific also.  You can go a year or so back into the git history of the code-in-docs article to find the old format documented.

Reply:
Found it:

```
[!code-csharp-interactive[](path/from/repo/root/Example.cs)]
```

Reply:
Just FYI that format doesn't work for all kinds of interactive options, that's why I had to convert to the new format.
That format defaults to interactive="try-dotnet", so you're losing information if you convert.